.travis.yml

```diff
@@ -22,8 +22,10 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE=partner_multi_relation
```

Comment:
OK, thanks for the information. The only problem with this technique is that runbot only installs the first declared module (in this case, partner_multi_relation), and not the rest. If we continue splitting this, better to put first the EXCLUDE line to have all the modules except this one.